<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QService</class>
 <widget class="QDialog" name="QService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Origen:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtOrigen"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Destino:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtDestino"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Precio:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Veh√≠culo:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListWidget" name="listAutos"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dsbPrecio">
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>2.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="dteFecha"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
